### PR TITLE
A new home

### LINKED ISSUE #105: Update stars count on Homepage
On the main callout on the new homepage we have the number of stars that our repo has - we should update this some JS on page load so it's always up to date

### PR DESCRIPTION
**Pull request by ptoffy**

### CODE REVIEW

File: stack.yml
Comment (0xTim) [inline stack.yml:1]: We still want this since the stack is the same

File: README.md
Comment (0xTim) [inline README.md:1]: Keep the README with updates

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// swift-tools-version:5.5
```
Comment (0xTim) [inline Package.swift]: Bump this to 5.10

**Review by MahdiBM — COMMENTED** [review]
Nice deployment preview we have there.
Looks good.

One inaccuracy is that @gwynne pruned a lot of the Discord members so users inactive past 1 month or something (at that time) got kicked and we now have only 4K members.

Though the "online members" number is still the same and the ratio should be even higher ... so maybe we could focus on that "online members" number.

File: Resources/static/js/scrollNavbar.js
Comment (0xTim) [inline Resources/static/js/scrollNavbar.js:1]: Does this script not live in the design repo?

File: Sources/Website/ComingSoonPage.swift [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            }.class("main-page-callout-buttons d-flex align-items-center justify-content-center flex-column flex-lg-row")
+
+            // Code sample
+            Div {
+                let html = """
+                import Vapor
+
+                let app = try await Application.make(.detect())
+
+                app.get("showcase") { req in
```
Comment (0xTim) [inline Sources/Website/ComingSoonPage.swift]:
```suggestion
                app.get("\(title.lowercase)") { req in
```

**Review by 0xTim — CHANGES_REQUESTED** [review]
Some things I've noticed:

* Leaf link points to 4.0 we should fix that
* The compiler error has a `*` - I think it should be `'`
* The showcase section links aren't implemented. See full list should point to the showcase page for example and the scroll buttons I'm assuming don't work as we don't have any more to show? (They work on mobile)
* Become a supporter link does nothing
* The sponsors section on mobile is cut off, we should show the scroll buttons below a certain size or wrap them in a flex box so they're on different lines (can't remember what the design had)

Looking great though!

File: Resources/static/js/scrollNavbar.js
Comment (ptoffy) [reply Resources/static/js/scrollNavbar.js:1]: It does but I wanted to bring in all the scripts we use on the main site over here. I can just point it to the design if you prefer, but now if we need to change something we don't need to do it there

Comment (0xTim) [reply Resources/static/js/scrollNavbar.js:1]: Ah yeah fair, we won't use this on other sites so it's all fine

**Review by 0xTim — APPROVED** [review]
Looks good to go! Just need to fix the showcase link

**Pull request merged**